### ISSUE TITLE
I have a question.

### ISSUE THREAD

**Issue by arserina** [marcoaaguiar/poe2-tree#49]
I don't know how to use github well, so I don't know where to contact, so I'm leaving this message.
Thank you for creating a good page before the early start.
Can you possibly create an executable file for this page?
Since I use a foreign language, I want to translate and edit it.
I can do the translation and editing.
I wonder if it's possible.

**Comment by Bolado** [comment]
> I don't know how to use github well, so I don't know where to contact, so I'm leaving this message. Thank you for creating a good page before the early start. Can you possibly create an executable file for this page? Since I use a foreign language, I want to translate and edit it. I can do the translation and editing. I wonder if it's possible.

Built-ins page translators found on Chrome or Edge works quite well on the page, it will translate the nodes as well. 

Other than that I think the easiest way would be for you to clone the repository, translate the texts found on the `src/lib/data/nodes.json` file and then run locally the project doing `npm install` and then `npm run dev` on the root folder. 

If you are totally new to programming, you just have to ask gpt or something on how to clone a repository, and then run a svelte project locally.

**Comment by marcoaaguiar** [comment]
I believe that it wouldn't be too difficult to implement a way to choose between different translations having English as a fallback. If anyone wants to give it a try I would happily accept the PR

**Comment by mortslhawmit** [comment]
I dont think this would be to difficult to add eighter and ill make a concept in a PR and we can discuss if we have a better approach
